src/Image/Controllers/ImageController.php

```diff
@@ -79,6 +80,20 @@ public function handleImage(UploadedFile $uploadedFile): NotEmptyString
         return new NotEmptyString($filename);
     }
 
+    /**
+     * @param string $fileName
+     * @return bool
+     * @throws \League\Flysystem\FileNotFoundException
+     */
+    public function delete(string $fileName): bool
```

Comment:
Deze methode moet eigenlijk geen return hebben, mag gewoon `void` zijn. Want als de file er niet is, is de file deleted :)